yt_dlp/extractor/lsm.py

```diff
@@ -267,12 +269,27 @@ def _real_extract(self, url):
 
         data = self._search_nuxt_data(
             self._fix_nuxt_data(webpage), video_id, context_name='__REPLAY__')
+        playback_type = traverse_obj(data, ('playback', 'type'))
```

Comment:
this can and should be fatal imo
```suggestion
        playback_type = data['playback']['type']
```